<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FrapTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Diffusion Constant</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>510</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabResults">
     <attribute name="title">
      <string>Results</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>711</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabImage">
     <attribute name="title">
      <string>Linear Fit</string>
     </attribute>
     <widget class="PlPlotWidget" name="pl_widget" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>711</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport_Images">
     <property name="title">
      <string>Import Images</string>
     </property>
     <addaction name="actionBackground"/>
     <addaction name="actionClosed_Aperature"/>
     <addaction name="actionImage_Stack"/>
    </widget>
    <addaction name="menuImport_Images"/>
    <addaction name="actionExport_Data"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun_Experiment"/>
    <addaction name="actionShow_Graph"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBackground"/>
   <addaction name="actionClosed_Aperature"/>
   <addaction name="actionImage_Stack"/>
   <addaction name="separator"/>
   <addaction name="actionRun_Experiment"/>
   <addaction name="actionShow_Graph"/>
  </widget>
  <action name="actionBackground">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/image_add.png</normaloff>:/images/image_add.png</iconset>
   </property>
   <property name="text">
    <string>Background</string>
   </property>
   <property name="toolTip">
    <string>Open Background</string>
   </property>
  </action>
  <action name="actionClosed_Aperature">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/image_add.png</normaloff>:/images/image_add.png</iconset>
   </property>
   <property name="text">
    <string>Closed Aperature</string>
   </property>
  </action>
  <action name="actionImage_Stack">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/images.png</normaloff>:/images/images.png</iconset>
   </property>
   <property name="text">
    <string>Image Stack</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExport_Data">
   <property name="text">
    <string>Export Data</string>
   </property>
  </action>
  <action name="actionRun_Experiment">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/control_play_blue.png</normaloff>:/images/control_play_blue.png</iconset>
   </property>
   <property name="text">
    <string>Run Experiment</string>
   </property>
  </action>
  <action name="actionShow_Graph">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/chart_curve_go.png</normaloff>:/images/chart_curve_go.png</iconset>
   </property>
   <property name="text">
    <string>Show Graph</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlPlotWidget</class>
   <extends>QWidget</extends>
   <header>plplotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
